Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -217,6 +218,20 @@ out _analysisProcess
             return null;
         }
 
+        /// <summary>
+        /// Raised when any file has a new analysis.
+        /// </summary>
+        public override event EventHandler<AnalysisCompleteEventArgs> AnalysisComplete;
+
+        public override IEnumerable<string> Files {
+            get {
+                return _projectFiles.Keys;
```

Comment:
It's a concurrent dictionary

---
In reply to: [68104072](https://github.com/Microsoft/PTVS/pull/1337#discussion_r68104072) [](ancestors = 68104072)